### PR TITLE
Point ironic-inspector to the ironic image in quay

### PR DESCRIPTION
**Pull request by elfosardo**
After ironic-inspector merges with ironic image, we need to change the
reference of the ironic-inspector container to point to the ironic image
in quay.

This should happen after metal3-io/ironic-image#253

### CODE REVIEW

**Review by dtantsur — APPROVED** [review]

**Comment by kashifest** [comment]
The tests are apparently failing. The inspector has the following log entries

```
2021-05-10 11:35:46.126 1 ERROR ironic_inspector.conductor.manager [-] The periodic ironic_inspector.conductor.manager.sync_with_ironic failed with: Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/futurist/periodics.py", line 293, in run
    work()
  File "/usr/lib/python3.6/site-packages/futurist/periodics.py", line 67, in __call__
    return self.callback(*self.args, **self.kwargs)
  File "/usr/lib/python3.6/site-packages/futurist/periodics.py", line 181, in decorator
    return f(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/ironic_inspector/conductor/manager.py", line 226, in sync_with_ironic
    ironic = ir_utils.get_client()
  File "/usr/lib/python3.6/site-packages/ironic_inspector/common/ironic.py", line 62, in get_client
    session=session, oslo_conf=CONF).baremetal
  File "/usr/lib/python3.6/site-packages/openstack/service_description.py", line 87, in __get__
    proxy = self._make_proxy(instance)
  File "/usr/lib/python3.6/site-packages/openstack/service_description.py", line 223, in _make_proxy
    data = proxy_obj.get_endpoint_data()
  File "/usr/lib/python3.6/site-packages/keystoneauth1/adapter.py", line 312, in get_endpoint_data
    return self.session.get_endpoint_data(auth or self.auth, **kwargs)
  File "/usr/lib/python3.6/site-packages/keystoneauth1/session.py", line 1260, in get_endpoint_data
    return auth.get_endpoint_data(self, **kwargs)
  File "/usr/lib/python3.6/site-packages/keystoneauth1/plugin.py", line 381, in get_endpoint_data
    **kwargs)
  File "/usr/lib/python3.6/site-packages/keystoneauth1/plugin.py", line 144, in get_endpoint_data
    discover_versions=discover_versions)
  File "/usr/lib/python3.6/site-packages/keystoneauth1/discover.py", line 1061, in get_versioned_data
    max_version=max_version)
  File "/usr/lib/python3.6/site-packages/keystoneauth1/discover.py", line 1140, in _set_version_info
    discover_versions=discover_versions)
  File "/usr/lib/python3.6/site-packages/keystoneauth1/discover.py", line 1250, in _run_discovery
    "Version requested but version discovery document was not"
keystoneauth1.exceptions.discovery.DiscoveryFailure: Version requested but version discovery document was not found and allow_version_hack was False
: None: None
2021-05-10 11:35:46.126 1 ERROR ironic_inspector.conductor.manager NoneType: None
2021-05-10 11:35:46.126 1 ERROR ironic_inspector.conductor.manager 
```
Wait until the tests finish and you will get the logs as a zip file. 

/lgtm cancel

**Comment by elfosardo** [comment]
@kashifest  weird, it was successful last week

File: vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml [hunk 1]
```diff
@@ -246,7 +246,7 @@
 
   - name: Upgrade ironic-inspector image based containers
```
Comment (furkatgofurov7) [inline vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml:247]:
```suggestion
  - name: Upgrade ironic image based containers 
```

**Comment by furkatgofurov7** [comment]
@elfosardo hi! We can take this in hopefully once we merge [baremetal-operator/pull/886](https://github.com/metal3-io/baremetal-operator/pull/886) and [ironic-image/pull/264](https://github.com/metal3-io/ironic-image/pull/264). Also, was checking a bit and found out that there are couple of places where it needs a cleanup in project-infra repo too, do you want to take care of it too otherwise just let me know I can take care of it. Thanks.

**Comment by elfosardo** [comment]
> @elfosardo hi! We can take this in hopefully once we merge [baremetal-operator/pull/886](https://github.com/metal3-io/baremetal-operator/pull/886) and [ironic-image/pull/264](https://github.com/metal3-io/ironic-image/pull/264). Also, was checking a bit and found out that there are couple of places where it needs a cleanup in project-infra repo too. Do you want to take care of it too otherwise just let me know I can take care of it. Thanks.

@furkatgofurov7 thanks for the comments! I have a change ready for project-infra but forgot to submit the PR! Going to do it today.

**Comment by elfosardo** [comment]
@furkatgofurov7 project-infra change https://github.com/metal3-io/project-infra/pull/193

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Pull request merged**